judge/models/problem_data.py

```diff
@@ -30,6 +30,11 @@ def problem_directory_file(data, filename):
     ('linecount', _('Line-by-line')),
 )
 
+HINTS = (
+    ('unicode', _('Unicode')),
+    ('none', _('NONE'))
```

Comment:
We also have `nobigmath` (https://github.com/DMOJ/java-sandbox-agent); `nobuf` isn't necessary because there's a language-agnostic `unbuffered` option that should be used instead.